### ISSUE TITLE
Bundles: Unable to delete launcher and dependency packages

### ISSUE THREAD

**Issue by martastain** [ynput/ayon-frontend#734]
When an incorrect version of a dependency package or launcher is uploaded, there is no way to delete them from the frontend. 

`[DELETE] /api/desktop/installers/{filename}` and `[DELETE] /api/desktop/dependencyPackages/{filename}` endpoints exists, so it should be just a frontend missing the right buttons/subpage

- New tab on upload dialog that switches between manage and upload (default always upload).
- Multi-selection of items with delete button.
- Each column should be sortable.
- Size should format to reasonable label `bytes` `kb` `mb` `gb`.
- Use tanstack table.
![Image](https://github.com/user-attachments/assets/7e8d807e-7bdb-4128-b234-08e0784e1362)

**Comment by Innders** [comment]
Can you overwrite them?

**Comment by martastain** [comment]
> Can you overwrite them?

Yes. To overwrite the manifest record, you need to specify `?force=true`, but the actual upload is PUT and it just replaces by default.

**Comment by Innders** [comment]
So other than being nice to be able to cleanup old/wrong uploads, this isn't actually required?

**Comment by Innders** [comment]
Pending design...
